src/github.com/matrix-org/dendrite/clientapi/error/error.go

```diff
@@ -0,0 +1,62 @@
+package error
```

Comment:
More to the point how will people use this when the package name clashes with the builtin error type ?